app/components/Navigation.js

```diff
@@ -96,28 +88,28 @@ class Navigation extends Component {
     const { user } = this.props;
     return (
       <Navbar
-        bg="dark"
+        id="navbar"
         style={{
           height: '7rem',
-          backgroundColor: 'black',
```

Comment:
I actually didn't mean to change the background color in Navbar. Could you keep it the same as the default setting?

Reply:
Done